lib/spack/spack/test/concretize_preferences.py

```diff
@@ -105,28 +103,16 @@ def test_preferred_variants_from_wildcard(self):
         update_packages("multivalue-variant", "variants", "foo=bar")
         assert_variant_values("multivalue-variant foo=*", foo=("bar",))
 
-    def test_preferred_compilers(self):
+    @pytest.mark.parametrize(
+        "compiler_str,spec_str",
+        [("gcc@4.5.0", "mpileaks"), ("clang@12.0.0", "mpileaks"), ("gcc@4.5.0", "openmpi")],
```

Comment:
@alalazo this will fail on `aarch64`

Reply:
Can we use `gcc@10.2.1` instead?

Reply:
Sure, whatever passes without using `os=` doesn't trigger the slowdon.